CHANGELOG.md

```diff
@@ -1,7 +1,659 @@
-# CHANGELOG
+# What's New
 
 All notable changes to NiMARE releases are documented in this page.
 
 ## [Unreleased](https://github.com/neurostuff/NiMARE/compare/0.0.12rc2...HEAD)
 
-<!-- New Release notes will be placed here automatically -->
\ No newline at end of file
+<!-- New Release notes will be placed here automatically -->
+
+## 0.0.12rc3 (2022-03-21)
+
+Just a quick release for debugging.
+
+<!-- Release notes generated using configuration in .github/release.yml at main -->
+
+## What's Changed
+### Other Changes
+* Document other meta-analysis tools outside our ecosystem by @tsalo in https://github.com/neurostuff/NiMARE/pull/654
+* Reorganize and streamline examples by @tsalo in https://github.com/neurostuff/NiMARE/pull/656
+* Convert CBMAEstimator method to function by @tsalo in https://github.com/neurostuff/NiMARE/pull/658
+
+
+**Full Changelog**: https://github.com/neurostuff/NiMARE/compare/0.0.12rc2...0.0.12rc3
+
+## 0.0.12rc2 (2022-03-04)
+
+This release candidate includes a number of changes. We have added a new `FocusCounter` diagnostic tool for characterizing results of coordinate-based meta-analyses, and have added cluster-level correction to the MKDAChi2 Estimator's Monte Carlo FWE correction method. We have also made some changes to the documentation formatting that should hopefully make the NiMARE documentation more informative and easier to read.
+
+<!-- Release notes generated using configuration in .github/release.yml at main -->
+
+## What's Changed
+### 🛠 Breaking Changes
+* Replace multiprocessing with joblib for parallelization and change n_cores default to 1 by @tsalo in https://github.com/neurostuff/NiMARE/pull/597
+* Incorporate joblib into ALESubtraction and fix SCALE docstring by @tsalo in https://github.com/neurostuff/NiMARE/pull/641
+### 🎉 Exciting New Features
```

Comment:
I noticed the issue in the [previous docs build](https://nimare--635.org.readthedocs.build/en/635/changelog.html#breaking-changes). I only see the first heading `### 🛠Breaking Changes`, the other ones (`### 🎉 Exciting New Features`, `### 🐛 Bug Fixes`, and `### Other Changes`) are mixed in the text.
If that is the case, it may be an issue for the automatic changelog, since it will copy the body of the release notes as it is (without blank lines between `###` headings).

Reply:
I don't know how to fix this though... the template doesn't seem to let us control spacing.

Reply:
Hopefully the auto-generated version will look okay. I'll merge this once CI passes and then I'll make a release (0.0.12rc4 😬).